pcsx2/gui/AppMain.cpp

```diff
@@ -563,6 +586,27 @@ void Pcsx2App::LogicalVsync()
 		}
 	}
 
+	if (g_Conf->GSWindow.IsToggleAspectRatioSwitch) {
+		if (EnableFMV == 1) {
```

Comment:
Want me to update that for the "Switch to GSdx software" gamefix too? That's where I'm borrowing it from.